examples/with-ant-design/package.json

```diff
@@ -6,8 +6,9 @@
     "start": "next start"
   },
   "dependencies": {
+    "@ant-design/cssinjs": "^1.2.0",
     "@ant-design/icons": "^4.8.0",
-    "antd": "^5.0.0",
+    "antd": "^5.1.0",
     "next": "latest",
```

Comment:
```json
"antd": "^5.3.1"
"@ant-design/cssinjs": "^1.6.1"
"next": "^13.2.4"
```
i updated antd and next unstyled component issue exist again
https://github.com/ant-design/ant-design/issues/39891